locales/sq_AL/LC_MESSAGES/duckduckgo.po

```diff
@@ -325,28 +326,28 @@ msgstr "Përgjigjet në shumicën e pyetjeve të DuckDuckGo, si:"
 
 msgctxt "video-duration"
 msgid "Any Duration"
-msgstr ""
+msgstr "Çdo kohëzgjatje "
```

Comment:
Heads up, you added an extra space at the end of the string here and in the next 4 strings. This can cause problems in some cases, like layout misalignment and such. Please, remove them. 🙂 
```suggestion
msgstr "Çdo kohëzgjatje"
```